hack/charts/installer/rolebased/README.md

```diff
@@ -1,5 +1,49 @@
-things that need to be converted to helm:
-* AWS_DEFAULT_SAGEMAKER_ENDPOINT (TODO)
+# Amazon SageMaker Operator for Kubernetes Role-Based Installer
 
-Notes:
-* `kustomize build config/helm` -> has names that are actually variable references. We then replace those using helm.
+Amazon SageMaker Operator for Kubernetes Helm charts for role-based installation
+
+## Prerequisites
```

Comment:
Love this little readme! Consider adding link to ReadTheDocs documentation on installation.